exercises/00-init/src/server.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+
+
+const app = express()
+app.use(bodyParser.json())
+
+const users = [
+]
+
+app.get('/users', function (req, res) {
+    res.send(JSON.stringify(users))

```

Comment:
Express even has a convenient `json()` response method on the `res` object, so this could be
```
res.json(users)
```